### DISCUSSION TITLE
Crash in profile

### DISCUSSION THREAD

**Discussion by abobafett-dev** [community/community#24066] [Apps, API and Webhooks]
<p>iPhone SE</p>
<p>Below the avatar is the following button.  There is 1 organization on the list.  I click on the button, a list of organizations is displayed.  I click on the organization - the application crashes.  After that, when you click on the following, the application starts to crash.  After cleaning the cache, everything starts again.</p>

**Comment by jfullstackdev** [comment]
<p>is this GitHub Mobile on iPhone ? you can provide screenshots too</p>

**Comment by abobafett-dev** [comment]
<p>Yea, that GitHub Mobile Version 1.57.0 (62668849),<br>
on iPhone SE - ios 14.7.1</p>
<p>I click on “following”<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/5/458fa3d0807f2a3f4570d90dfdac7a007f863a83.png" data-download-href="/uploads/short-url/9VmGW4Nmkzes0GcSM96HrNDLtVF.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/77672564/181081787-f74fe372-fa70-4ac1-a84d-9086d2eb4844.png" alt="" data-base62-sha1="9VmGW4Nmkzes0GcSM96HrNDLtVF" width="282" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/5/458fa3d0807f2a3f4570d90dfdac7a007f863a83_2_282x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/5/458fa3d0807f2a3f4570d90dfdac7a007f863a83_2_423x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/5/458fa3d0807f2a3f4570d90dfdac7a007f863a83.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/5/458fa3d0807f2a3f4570d90dfdac7a007f863a83_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename"></span><span class="informations">486×861 136 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>next, this window opens<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/0/80e54b00ee01a78508ed1c0a2451f1c877220624.png" data-download-href="/uploads/short-url/iognciEzPcvC1ubx7rqrl7mDI5S.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/77672564/181081788-c0dcc3fa-76a0-40af-9405-05115360f12f.png" alt="" data-base62-sha1="iognciEzPcvC1ubx7rqrl7mDI5S" width="283" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/0/80e54b00ee01a78508ed1c0a2451f1c877220624_2_283x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/0/80e54b00ee01a78508ed1c0a2451f1c877220624_2_424x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/0/80e54b00ee01a78508ed1c0a2451f1c877220624.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/0/80e54b00ee01a78508ed1c0a2451f1c877220624_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename"></span><span class="informations">487×859 71.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>and then I click on the first organization in the list, after that the organization appears for a second and the application crashes</p>
<p>after that, the application starts to crash when I click on “following” in the first screenshot.</p>
<p>and if I clean the cache, then it all starts from the beginning.</p>
<p>I can also find the right organization through the search and log in to it without any problems.</p>

**Comment by abobafett-dev** [comment, marked as answer]
<p>It looks like this problem is solved with one of the patches.</p>
